beetsplug/lyrics.py

```diff
@@ -171,8 +173,15 @@ def _encode(s):
             s = s.encode('utf8', 'ignore')
         return urllib.quote(s)
 
-    def build_url(self, artist, title):
-        return self.URL_PATTERN % (self._encode(artist.title()),
+    def build_url(self, artist, album, title, url_format=None):
```

Comment:
Perhaps we should move the calls to `.lower()` and `.title()` to the clients of `build_url` instead of here.